wiki/Service/Video.md

```diff
@@ -0,0 +1,48 @@
+`Core` <!-- category start --><!-- category end -->
+
+The Video service intakes a Kaltura player <sup>[[r1](#resources)]</sup> `videoId` and fetches the video and associated meta data from Kaltura Media Center.
+
+<hr width="40%" />
+
+<!-- toc start open="true" --><!-- toc end -->
+
+<details open="true">
+  <summary><strong>Dependencies</strong> (<!-- dependencyCount start --><!-- dependencyCount end -->)</summary><br />
+
+- [[Kaltura API]] <sup>[[r1](#resources)]</sup>
+
+<br />
+</details>
+
+<!-- usedby start open="true" --><!-- usedby end -->
+
+<!-- backlinks start open="true" --><!-- backlinks end -->
+
+<a name="resources"></a>
+<details open="true">
+  <summary><strong>Resources</strong></summary><br />
+
+- r1: [Kaltura player](http://player.kaltura.com/docs/api)
+
+<br />
+</details>
+
+<hr width="40%" />
+
+## Input
+
+| Data name | Type |
+| --------- | ---- |
+| `videoID` | Number |
+
+## Output
+
+| Data name | Type |  
+| --------- | ---- |
+| Name | Text |
+| Description | Text |
+| Duration | Number |
+| Poster image | Image |
```

Comment:
I'm hesitant to rush something like this out the door. Is there a reason we need this page today? I like the red links because it's transparent that we haven't done this one yet.